bindings/c/CMakeLists.txt

```diff
@@ -6,8 +6,11 @@ set(vw_c_impl_files
   src/placeholder.cc
 )
 
+# In order to reuse the error definition data in the C API we must make the error_data.h header available here.
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/../../vowpalwabbit/error_data.h include/vw/experimental/error_data.h COPYONLY)
+
 add_library(vw_c_api ${vw_c_public_headers} ${vw_c_impl_files})
-target_include_directories(vw_c_api PUBLIC include/)
+target_include_directories(vw_c_api PUBLIC include/ ${CMAKE_CURRENT_BINARY_DIR}/include/)
```

Comment:
are the `include` and `{CMAKE_CURRENT_BINARY_DIR}/include` dirs different?

Reply:
Yes, when we configure file to copy the `error_data.h` file, it goes into the binary dir and not the source dir.